svgtypewriter.d.ts

```diff
@@ -210,3 +210,5 @@ declare namespace SVGTypewriter.Measurers {
         reset(): void;
     }
 }
+
+declare module "SVGTypewriter" { export = SVGTypewriter; }
```

Comment:
it should be called `"svg-typewriter"` to match up with the bower name (and we should rename it in package.json)